draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
+             <t>
+               When an answerer receives in offer that contains an 'identity' attribute, the answerer can use the
+               the attribute information to contact the IdP, and verify the identity of, the peer. If the answerer 
+               does not accept the attribute value, or if the identity verification fails, it MUST reject the offer.
+             </t>
+             <t>
+               When an answerer sends an answer, in order to provide its identity assertion to the peer, it MUST include an 
+               'identity' attribute in the answer. In addition, it MUST include 'fingerprint' attributes
+               for each fingerprint that have been used to assert the identity.
+             </t>
+             <t>
+               The answerer can include an 'identity' attribute in the answer independently of whether the 
+               corresponding offer contained an identity attribute or not.
+             </t>
+            </section>
+          </section>
+	      <section title="Modifying Identity Assertion within Session Description" anchor="sec.sdp-id-attr-oa-mod">
+	        <t>
+              When an endpoint has included its identity assertion in a previous offer or answer, and when the endpoint later in session
+              sends an offer or answer, the endpoint can either keep the identity assertion value unchanged, modify the value
+              or revoke the identity assertion. This section describes the procedures for that.
+            </t>
+            <t>
+              NOTE: Specific usages might restrict the modification and/or revoking of the identity assertion within a session.
+	        </t>
+            <section title="Offer Procedures" anchor="sec.sdp-id-attr-oa-mod-off">
+              <t>
+                When the offerer sends an offer, if the offerer wants to maintain the identity assertion value it previously provided (in an offer
+                or answer) as part of a successful offer/answer transaction within the session, it MUST include an identity attribute with the previously provided 
+                attribute value in the offer.
+              </t>
+              <t>
+                If the offerer wants to revoke its identity assertion from the session, it MUST NOT include an identity attribute in the offer.
+              </t>
+              <t>
+                If the offerer wants to modify its identity assertion within the session, it MUST include an
+                'identity' attribute value with the new identity assertion in the offer. In addition, it MUST include 'fingerprint' attributes
+                for each fingerprint that have been used to assert the identity.
+              </t>
+              <t>
+                When the offerer receives an answer that contains an 'identity' attribute, the offerer can use
+                the attribute information to contact the IdP, and verify the identity of, the peer. If the offerer does not
```

Comment:
If you keep this section instead of folding it in as Martin suggests, remove the final comma